Frontend/src/Components/GeneralData/Communication.js

```diff
@@ -0,0 +1,83 @@
+
+    const bgColor = !props.data?.solar_car_connection[0] ||
+            ((packetDelay.getHours() > 0) || (packetDelay.getMinutes() > 0) || (packetDelay.getSeconds() > 1)) ?
+            getColor("errorBg", colorMode) : null;
+
+
+    /**
+     * Get formatted delay between the current time and the most recent timestamp
+     * @returns {string} A formatted string containing the delay between the current time and the most recent timestamp
+     * @private
+     */
+    const _getFormattedPacketDelay = () => {
+        // Return formatted delay
+        return ((packetDelay.getHours() < 10) ? "0" + packetDelay.getHours() : packetDelay.getHours()) + ":"
+               + ((packetDelay.getMinutes() < 10) ? "0" + packetDelay.getMinutes() : packetDelay.getMinutes()) + ":"
+               + ((packetDelay.getSeconds() < 10) ? "0" + packetDelay.getSeconds() : packetDelay.getSeconds()) + "."
+               + ((packetDelay.getMilliseconds() >= 100) ? packetDelay.getMilliseconds() :
+                 ((packetDelay.getMilliseconds() >= 10) ? "0" + packetDelay.getMilliseconds() :
+                 "00" + packetDelay.getMilliseconds()));
+    };
+
+    return (
+        <Flex flex='auto' direction='column'>
+            <HeadingCell fontSize='2.2vh' label='Communication'/>
+            <Flex flex='inherit' direction='column' pl='2' justify='center' >
+                <CommsLabel
+                    boolean={props.data?.solar_car_connection[0]}
+                    label='Solar Car Connection'
+                />
+                <HStack>
+                    <Text fontSize='2vh' style={{ textIndent: 30 }}>&#160;Packet Delay: </Text>
+                    <Text fontSize='2vh' backgroundColor={bgColor}>{_getFormattedPacketDelay()}</Text>
+                </HStack>
+                <CommsLabel
+                    boolean={props.data?.mainIO_heartbeat[0]}
+                    label='Main IO Heartbeat'
+                />
+                <Box h='1vh' />
+                <Text fontSize='2vh'>CAN Heartbeats:</Text>
+                <Flex direction='column'>
+                    <Flex>
+                        <Box w='3vh' />
+                        <CommsLabel
+                            boolean={props.data?.bms_can_heartbeat[0]}
+                            label='BMS'
+                        />
+                    </Flex>
+                    <Flex>
+                        <Box w='3vh' />
+                        <CommsLabel
+                            boolean={props.data?.mcc_can_heartbeat[0]}
+                            label='MCC'
+                        />
+                    </Flex>
+                    <Flex>
+                        <Box w='3vh' />
+                        <CommsLabel            
+                            boolean={props.data?.mppt_can_heartbeat[0]}
+                            label='MPPT'
+                        />
+                    </Flex>
```

Comment:
```suggestion
                    <CommsLabel
                        indent={true}
                        boolean={props.data?.bms_can_heartbeat[0]}
                        label='BMS'
                    />
                    <CommsLabel
                        indent={true}
                        boolean={props.data?.mcc_can_heartbeat[0]}
                        label='MCC'
                    />
                    <CommsLabel
                        indent={true}
                        boolean={props.data?.mppt_can_heartbeat[0]}
                        label='MPPT'
                    />
```
Instead of having a separate Box and Flex surrounding CommsLabel, let's add an `indent` prop that conditionally includes the Box inside of CommsLabel.